### PR TITLE
Make `metric.getName()` into `metric.name`

### PR DESCRIPTION
**Pull request by MStoykov**
The original change was proposed in https://github.com/k6io/k6/issues/2053 and implemented in https://github.com/k6io/k6/pull/2058

### CODE REVIEW

**Review by na-- — APPROVED** [review]

File: js/modules/k6/metrics/metrics.go [hunk 1]
```diff
@@ -65,7 +65,18 @@ func newMetric(ctxPtr *context.Context, name string, t stats.MetricType, isTime
 	}
 
 	rt := common.GetRuntime(*ctxPtr)
-	return common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr), nil
+	binded := common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr)
```
Comment (na--) [inline js/modules/k6/metrics/metrics.go:68]:
```suggestion
	bound := common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr)
```
https://www.oxfordlearnersdictionaries.com/definition/american_english/bound_1

File: js/modules/k6/metrics/metrics.go [hunk 2]
```diff
@@ -65,7 +65,18 @@ func newMetric(ctxPtr *context.Context, name string, t stats.MetricType, isTime
...
+	o := rt.NewObject()
+	err := o.DefineAccessorProperty("name", rt.ToValue(func() interface{} {
+		return name
+	}), nil, goja.FLAG_FALSE, goja.FLAG_TRUE)
+	if err != nil {
+		return nil, err
+	}
+	if err = o.Set("add", rt.ToValue(binded["add"])); err != nil {
```
Comment (na--) [inline js/modules/k6/metrics/metrics.go:76]:
```suggestion
	if err = o.Set("add", rt.ToValue(bound["add"])); err != nil {
```

File: js/modules/k6/metrics/metrics.go [hunk 3]
```diff
@@ -65,7 +65,18 @@ func newMetric(ctxPtr *context.Context, name string, t stats.MetricType, isTime
 	}
 
 	rt := common.GetRuntime(*ctxPtr)
-	return common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr), nil
+	binded := common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr)
+	o := rt.NewObject()
+	err := o.DefineAccessorProperty("name", rt.ToValue(func() interface{} {
```
Comment (imiric) [inline js/modules/k6/metrics/metrics.go:70]: This could be `DefineDataProperty` instead, no? Since we don't need to call a function for it.

Comment (MStoykov) [reply js/modules/k6/metrics/metrics.go]: resolved in 20c22d9

**Review by na-- — APPROVED** [review]
LGTM, though please squash when/before merging

**Review by imiric — APPROVED** [review]

**Pull request merged**